File-Formats/XlsIO/Improving-Performance.md

```diff
@@ -335,3 +335,16 @@ validation.EndUpdate()
 {% endhighlight %}
 {% endtabs %}  
 
+## Performance and Memory benchmark
```

Comment:
1. Add separate headings as Performance Benchmark and Memory Consumption Benchmark.
2. Add actual HTML table for showing the benchmark
3. Link Github sample used to take this benchmark.